pkg/model/crd.go

```diff
@@ -507,6 +507,21 @@ func (r *CRD) UpdateConditionsCustomMethodName() string {
 	return resGenConfig.UpdateConditionsCustomMethodName
 }
 
+// CustomCheckRequiredFieldsMissingMethod returns custom check required fields missing method
+// as *string for custom resource, if specified in generator config
+func (r *CRD) CustomCheckRequiredFieldsMissingMethod(op *awssdkmodel.Operation) string {
```

Comment:
nit: Other methods don't add `Custom` prefix to this method, but use `Get` instead. `GetCheckRequiredFieldsMissingMethod` for example.

Reply:
I see `customimplementation` which inspired this is written as `GetCustomImplementation`. What do you think about `GetCustomCheckRequiredFieldsMissingMethod` ?

Reply:
Oh I see, yes!